src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -52,3 +62,27 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		jiraHost
 	});
 };
+
+const sortByDateString = (a, b) => {
+	return new Date(b).valueOf() - new Date(a).valueOf();
+};
+
+const getReposBySubscriptions = async (subscriptions: Subscription[], jiraHost: string, gitHubAppId: number | undefined, logger: Logger): Promise<RepositoryNode[]> => {
```

Comment:
This basically does the get repos call for each subscription... this is mostly fine..
of 100000 subscriptions only ~300 have four or more subs. There is a handful of 20's and 30's this should still work.... might be a bit poor performance though, so as a seperate story we can investigate introducing orgs option so that if we are overwhelmed with options we can refine to what they chose. Its an extra step which is lame but we are talking 0.1% of customers